qa/L0_shared_memory/shared_memory_test.py

```diff
@@ -29,124 +29,143 @@
 import sys
 
 sys.path.append("../common")
-
 import os
 import unittest
+from pathlib import Path
 
 import numpy as np
 import test_util as tu
 import tritonclient.grpc as grpcclient
 import tritonclient.http as httpclient
 import tritonclient.utils.shared_memory as shm
+import util
+from parameterized import parameterized
 from tritonclient import utils
 
 
 class SharedMemoryTest(tu.TestResultCollector):
-    def test_invalid_create_shm(self):
-        # Raises error since tried to create invalid system shared memory region
-        try:
-            shm_op0_handle = shm.create_shared_memory_region(
-                "dummy_data", "/dummy_data", -1
-            )
-            shm.destroy_shared_memory_region(shm_op0_handle)
-        except Exception as ex:
-            self.assertTrue(str(ex) == "unable to initialize the size")
+    # Constant members
+    shared_memory_test_client_log = Path(os.getcwd()) / "client.log"
+    model_dir_path = Path(os.getcwd()) / "models"
+    model_source_path = Path(os.getcwd()).parents[0] / "python_models/add_sub/model.py"
```

Comment:
Future follow-up, but might be nice to have some kind of `utils.relative_path([path, to, thing])`.

ex maybe something like this:
```python
model_dir_path = utils.relative_path("models")
model_source_path = utils.relative_path("..", "python_models", "add_sub", "model.py")
```